src/GravitationalClustering.js

```diff
@@ -1,38 +1,21 @@
 import Random from 'random-js';
-
-var Particle = require('./Particle');
-var UnionFind = require('ml-disjoint-set');
-var Distance = require('ml-distance').distance;
-var randomInt = require('./Utils').randomInt;
+import ParticleCluster from './ParticleCluster';
+import Particle from './Particle';
+import UnionFind from 'ml-disjoint-set';
+import {distance as Distance} from 'ml-distance';
```

Comment:
you don't have to save the function. just allow the user to pass it back in the load method